osu.Game/Skinning/DefaultLegacySkin.cs

```diff
@@ -20,7 +20,7 @@ public DefaultLegacySkin(IResourceStore<byte[]> storage, AudioManager audioManag
                 new Color4(242, 24, 57, 255)
             );
 
-            Configuration.LegacyVersion = 2.0m;
+            Configuration.LegacyVersion = 2.7m;
```

Comment:
An inline comment can be added here pointing out why `LATEST` const isn't used instead.

Reply:
i think it's pretty obvious. DefaultLegacySkin is basically constants from stable. if it requires a comment then we might as well just use LATEST and rename it to LAST_VERSION.